.github/workflows/build.yml

```diff
@@ -26,6 +26,17 @@ jobs:
           uv venv
           uv sync --all-extras
 
+      - name: Replace connection string placeholder
+        run: |
+          originalfile="src/uipath/telemetry/_constants.py"
+          tmpfile=$(mktemp)
+          trap 'rm -f "$tmpfile"' EXIT
+
+          rsync -a --no-whole-file --ignore-existing "$originalfile" "$tmpfile"
+          envsubst '$CONNECTION_STRING' < "$originalfile" > "$tmpfile" && mv "$tmpfile" "$originalfile"
+        env:
+          CONNECTION_STRING: ${{ secrets.APPINS_CONNECTION_STRING }}
```

Comment:
Please discuss if we should have an API that returns the connection string (eg: if we decide to change it)

Reply:
Also please make sure that having an invalid key fails silently and doesn't block execution